code/datums/components/shell.dm

```diff
@@ -25,7 +25,7 @@
 	COOLDOWN_DECLARE(power_used_cooldown)
 
 	/// The maximum power that the shell can use in a minute before entering overheating and destroying itself.
-	var/max_power_use_in_minute = 20000
+	var/max_power_use_in_minute = (20000 KILO WATTS)
```

Comment:
```suggestion
	var/max_power_use_in_minute = 20 * STANDARD_CELL_CHARGE
```

So we don't get this issue again if someone decides to change STANDARD_CELL_CHARGE.

Reply:
ya thats smarter